pkgs/ffigen/test/native_objc_test/protocol_test.dart

```diff
@@ -128,13 +223,13 @@ void main() {
         final sel = registerName('voidMethod:');
         final signature = getProtocolMethodSignature(proto, sel,
             isRequired: false, isInstance: true);
-        final block = DartVoidMethodBlock.listener((Pointer<Void> p, int x) {
+        final block = VoidMethodBlock.listener((Pointer<Void> p, int x) {
```

Comment:
Should you have a test where a listener is consuming an objective-c instance?

Reply:
Once I fix https://github.com/dart-lang/native/issues/835, yeah.